### ISSUE TITLE
Rename WebRTC classes inside TwilioVideo

### ISSUE THREAD

**Issue by wenxin3262** [twilio/twilio-video-ios#18]
### Description

When installing TwilioVideo with another WebRTC framework together, Xcode compiler will complain about duplicate objc class names.

Our app is running fine since we are trying to use the smiliar version of the WebRTC that included in your library.

My request is change all WebRTC symbols to something unique to your framework. e.g. TwilioRTCXXX 

### Steps to Reproduce

1. Use quickstart project, and install another pod that is or uses WebRTC
2. Import the framework and run the app.

#### Code

add to podfile
```ruby
pod 'GoogleWebRTC'
```

#### Expected Behavior

No warnings coming from the compiler.

#### Actual Behavior

The compiler complains about duplicate WebRTC class names and will lead to indeterministic behavior on different platforms.

#### Reproduces How Often

100%

#### Logs

```
objc[4675]: Class RTCVideoFrame is implemented in both /private/var/containers/Bundle/Application/E00E00E0-9DA1-48C0-84B0-7F73D6D33AA7/XXX.app/Frameworks/WebRTC.framework/WebRTC (0x10b4e8058) and /private/var/containers/Bundle/Application/E00E00E0-9DA1-48C0-84B0-7F73D6D33AA7/XXX.app/Frameworks/TwilioVideo.framework/TwilioVideo (0x10750a4c8). One of the two will be used. Which one is undefined.

and two dozens of the same
```

### Versions

All

#### Video iOS SDK

2.3.0-preview1 via CocoaPods

#### Xcode

9.4.1

#### iOS Version

11.4.1

#### iOS Device

all

**Comment by ceaglest** [comment]
Hi @wenxin3262,

Thanks for filing an issue.

This conflict is something that we've wanted to address for a long time. I'm happy to report that we are currently making improvements to our CI process which will allow us to rename the conflicted `RTC` classes. If all goes to plan our next `2.3.0-preview2` release will include a version of WebRTC built with these changes.

I'll keep you updated as we roll this out.

Best,
Chris

**Comment by piyushtank** [comment]
@wenxin3262  Also, please note if you are running `TwilioVideo.framework` iOS 12 Beta, since `WebCore.framework` includes webrtc's Objective-C classes, due to the same issue of Objective-C class name conflicts, the runtime errors are thrown. see [[this](https://bugs.chromium.org/p/webrtc/issues/detail?id=9500)]. As @ceaglest mentioned that we are working on fixing it in our Video SDK and planning to have the fix included in `2.3.0-preview2`.

**Comment by piyushtank** [comment]
@wenxin3262 We have just released 2.3.0-preview2 where you use the Video SDK side by side with any other Chromium WebRTC based dependency in the same application.  [[Here](https://www.twilio.com/docs/video/changelog-twilio-video-ios-version-2x#230-preview2-aug-2-2018)] are the 2.3.0-preview2 changelogs. 

Let us know if you have any questions.

Best,
Piyush

**Comment by ceaglest** [comment]
Now that [2.3.0](
https://www.twilio.com/docs/video/changelog-twilio-video-ios-version-2x#230-august-7-2018) has been live for some time and includes the prefixing changes I'm going to close this ticket. @wenxin3262 if you have any issues adopting the new releases please let us know.
